cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProfileHttpHandler.java

```diff
@@ -182,6 +182,10 @@ private void validateProvisionerProperties(ProfileCreateRequest request) throws
     Collection<ProvisionerPropertyValue> provisionerProperties = provisionerInfo.getProperties();
     if (provisionerProperties != null) {
       for (ProvisionerPropertyValue value : provisionerProperties) {
+        if (value == null) {
+          throw new BadRequestException("A provisioner property cannot be null. Please make sure it is " +
```

Comment:
I think it's better just to ignore it if null instead of making it an error.

Reply:
👍